scripts/bazel/build-packages.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Exit if any of the builds fails
+set -e
+
+# Source directories for packages to build with bazel and copy into dist/
+packages=(cdk lib cdk-experimental material-experimental material-moment-adapter)
```

Comment:
you could consider doing a bazel query here like we do in Angular.
Then the configuration is more local (eg. use a tag on each package you want to pick up)

Reply:
Done, I think. Not sure if it's exactly what you had in mind.